meta/send.rb

```diff
@@ -224,6 +224,22 @@
 end
 
 Mutant::Meta::Example.add do
+  source 'foo.values_at(a, b)'
```

Comment:
@backus In the tests for #511 I noticed you used `self.` instead of `foo.`. Should we use one form for all new mutations consistently?

Also, it's probably outside of scope, but a future PR should probably make sure it's consistent across the board.